style.css

```diff
@@ -367,3 +370,139 @@ textarea {
   font-weight: 700;
   color: #172b4d;
 }
+
+/* RESPONSIVE */
+
+.only-desktop {
+  display: none;
+}
+
+@media only screen and (min-device-width: 480px) {
```

Comment:
- [ ] It is better if we set a `min-width` value to be in the range of `992px - 1100px`, this will allow us to set few media queries for the mobile version. Instead of having 3 media queries, we shall have 2 media queries.